CONTRIBUTING.md

```diff
@@ -141,6 +141,7 @@ linting is in order:
 ```
 poetry run bash scripts/format.sh
 poetry run bash scripts/lint.sh
+poetry run bash scripts/docstring.sh
 poetry run bash scripts/check-spelling.sh
```

Comment:
wonder whether this a *lot* for people to do & if we should consider consolidating some of these into a single command for the sake of external contributors, albeit keeping the individual scripts / commands for ourselves...)

Reply:
Great idea! I added a `run-ci-checks.sh` script that combines linting, docstring checks, and spell checks and adjusted the contributing guide accordingly.